File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L1/05/inputoutput.py
import math

# Get user input
radius = float(input("Enter the radius of the cylinder: "))
length = float(input("Enter the length of the cylinder: "))

# Calculate area and volume
area = math.pi * radius ** 2
volume = area * length

# Print results
print(f"The area of the cylinder is {area:.2f}")
print(f"The volume of the cylinder is {volume:.2f}")

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L6/05/inputoutput.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

class inputoutput {
  static rad: number;
  static panjang: number;

  static input(): void {
    rl.question('Enter the radius and length of a cylinder: ', (input) => {
      const [radius, length] = input.split(' ').map(Number);
      this.rad = radius;
      this.panjang = length;
      this.calculateAndPrint();
      rl.close();
    });
  }

  static hitArea(): number {
    return Math.pow(this.rad, 2) * 3.14159;
  }

  static calculateAndPrint(): void {
    const area = this.hitArea();
    const volume = area * this.panjang;
    console.log(`The area is ${area}`);
    console.log(`The volume of the cylinder is ${volume}`);
  }
}

inputoutput.input();

